### ISSUE TITLE
Page list

### ISSUE THREAD

**Issue by jestrux** [jestrux/nextiva#14]
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=5466%3A118611&mode=design&t=Ru7mn8JLPo9KsJf2-1
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=5466%3A118612&mode=design&t=Ru7mn8JLPo9KsJf2-1
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=5466%3A118613&mode=design&t=Ru7mn8JLPo9KsJf2-1
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=10196%3A43822&mode=design&t=Ru7mn8JLPo9KsJf2-1
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=5466%3A118614&mode=design&t=Ru7mn8JLPo9KsJf2-1
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=5854%3A38198&mode=design&t=Ru7mn8JLPo9KsJf2-1 (dark version)
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=5974%3A120443&mode=design&t=Ru7mn8JLPo9KsJf2-1 (dark version)
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=10261%3A511500&mode=design&t=Ru7mn8JLPo9KsJf2-1
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=11646%3A77135&mode=design&t=whVqRvlcZ9XZi6tP-1
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=11646%3A77130&mode=design&t=whVqRvlcZ9XZi6tP-1 (only light version)
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=10380%3A147373&mode=design&t=whVqRvlcZ9XZi6tP-1
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=5974%3A131727&mode=design&t=whVqRvlcZ9XZi6tP-1
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=9863%3A225865&mode=design&t=whVqRvlcZ9XZi6tP-1 (they want both dark & light to test)
- https://www.figma.com/file/cmBWPjsFEdLh1FaB1RGH2T/Nextiva-Website-Redesign-4.0?type=design&node-id=9863%3A294260&mode=design&t=whVqRvlcZ9XZi6tP-1 (light)

**Comment by walter** [comment]
I don't believe I'm the Walter you meant to mention. I'll unsubscribe, but you may want to mention them at the correct user name so they see this.

**Comment by jestrux** [comment]
Haha, it wasn't even meant to be a quote.

> I don't believe I'm the Walter you meant to mention. I'll unsubscribe, but you may want to mention them at the correct user name so they see this.
